src/createServer.js

```diff
@@ -1,8 +1,48 @@
 'use strict';
 
+const http = require('http');
+const fs = require('fs');
+const url = require('url');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  return http.createServer((req, res) => {
+    const normalizedUrl = new url.URL(req.url, `http://${req.headers.host}`);
+    const pathname = normalizedUrl.pathname;
+
+    if (pathname.includes('//')) {
+      res.statusCode = 404;
+      res.setHeader('Content-Type', 'text/plain');
+
+      return res.end('404 Not Found');
+    }
+
+    if (!pathname.startsWith('/file')) {
+      res.setHeader('Content-Type', 'text/plain');
+      res.statusCode = 400;
+
+      return res.end(
+        JSON.stringify('Invalid URL. To load files, use /file/<file_path>'),
```

Comment:
The `JSON.stringify` method is unnecessary here because the message is already a string. You can directly return the string: `'Invalid URL. To load files, use /file/<file_path>'`.